src/widgets/overview.js

```diff
@@ -1,74 +1,76 @@
 import { Op } from 'sequelize';
-import moment from 'moment';
+
 import {
   ActivityReport, ActivityRecipient, Grant, NonGrantee, sequelize,
 } from '../models';
 import { REPORT_STATUSES } from '../constants';
 
-/*
-  Widgets on the backend should only have to worry about fetching data in the format required
-  by the widget. In this case we return a single object but other widgets my require an array
-  (say for a time series). All widgets will need to honor the scopes that are passed in. All
-  that is required is that the scopes parameter is used as value for the `where` parameter (or
-  combined with [op.and] if the widget needs to add additional conditions to the query).
+export default async function dashboardOverview(scopes) {
+  const duration = await ActivityReport.findAll({
+    attributes: [
+      'duration',
+      'deliveryMethod',
+      'numberOfParticipants',
```

Comment:
Can we combine these two sql queries and sum/count these attributes the same way has reports and grants/non-grantees? I would think `duration` and `numberOfParticipants` just need a `SUM(..)` and `deliveryMethod` follows the pattern of the other three columns.

Reply:
Wouldn't inPerson require a separate or inline query given you need to add up the duration of only reports that have "in person" as their delivery method?

Reply:
another issue seems to be that some of the legacy data throws off the SUM
For example, 
```js
[sequelize.fn('SUM', sequelize.col('"ActivityReport".numberOfParticipants')), 'numParticipants'],
```
yields 769,802 whereas the way it is now, with the ```reduce``` is 76,944. Seems like at least some of the data is stored as a string.

A sum of the duration in the same way is also wildly inflated, although I'm not exactly sure why. Maybe it's the same issue. 

Not sure if they is a SQL forward way to write this instead of the JS solution I have.

Reply:
I think I remember what is going on. The `required: false` causes the joins to be left outer joins, which will give us duplicated activity reports. If a report has five recipients that report will be returned five times, which is messing with the results. There is probably a way to solve it in sql but it isn't worth it, I'd keep the two queries. Can you add a comment explaining the joining situation and why we can't have a single query? I'm almost positive this is the second time I've wonder about this, hopefully a comment will keep me from wasting everyones time a third time!

Reply:
Haha, will do. Not a time waster to think it through and get it right, IMO

Looks like adding the comment dismissed your review!